### PR TITLE
improve output of debugging APIs

### PR DESCRIPTION
**Pull request by jsteemann**
### Scope & Purpose

Improve output of debug APIs:
* improve date/time format for expire and start times
* include transaction id in query registry output
* add "queryid" to allowed structured logging parameters

- [ ] :hankey: Bugfix
- [x] :pizza: New feature
- [ ] :fire: Performance improvement
- [ ] :hammer: Refactoring/simplification

### Checklist

- [ ] Tests
  - [ ] **Regression tests**
  - [ ] C++ **Unit tests**
  - [ ] **integration tests**
  - [ ] **resilience tests**
- [ ] :book: CHANGELOG entry made
- [ ] :books: documentation written (release notes, API changes, ...)
- [ ] Backports
  - [ ] Backport for 3.12.0: -
  - [ ] Backport for 3.11: -
  - [ ] Backport for 3.10: -

#### Related Information

- [ ] Docs PR: 
- [ ] Enterprise PR:
- [ ] GitHub issue / Jira ticket:
- [ ] Design document:

### CODE REVIEW

**Review by mpoeter — APPROVED** [review]

File: arangod/Aql/RestAqlHandler.cpp [hunk 1]
```diff
@@ -67,6 +68,12 @@ RestAqlHandler::RestAqlHandler(ArangodServer& server, GeneralRequest* request,
   TRI_ASSERT(_queryRegistry != nullptr);
 }
 
+RestAqlHandler::~RestAqlHandler() {
+  if (_logContextQueryIdEntry) {
+    LogContext::Current::popEntry(_logContextQueryIdEntry);
```
Comment (neunhoef) [inline arangod/Aql/RestAqlHandler.cpp:73]: I see this `popEntry` here and the one in `shutdownExecute`. How to we prevent that both are executed when the query is finished? As far as I see the member `_logContextQueryIdEntry` is not set to `nullptr`...

**Review by neunhoef — APPROVED** [review]
LGTM, but please see my question in a comment. Probably, I missed something here, but I would like to understand this.

File: arangod/Aql/RestAqlHandler.cpp (hunk 1 above)
Comment (jsteemann) [reply arangod/Aql/RestAqlHandler.cpp:73]: `popEntry` does a check first, and also sets the entry to a nullptr. Therefore it is safe to call it multiple times on the same item.
Here is its implementation:
```
inline void LogContext::Current::popEntry(EntryPtr& entry) noexcept {
  if (entry._entry != nullptr) {
    auto& local = LogContext::controlBlock();
    TRI_ASSERT(entry._entry == local._logContext._tail);
    local._logContext.popTail(local._entryCache);
    entry._entry = nullptr;
  }
}
```

Comment (neunhoef) [reply arangod/Aql/RestAqlHandler.cpp:73]: Understood. Thanks!

**Pull request merged**